sycl/source/spirv_ops.cpp

```diff
@@ -24,18 +24,17 @@ __SYCL_EXPORT void __spirv_GroupWaitEvents(__spv::Scope Execution,
   (void)WaitEvents;
 }
 
-__SYCL_EXPORT void __spirv_ControlBarrier(__spv::Scope Execution,
-                                          __spv::Scope Memory,
-                                          uint32_t Semantics) noexcept {
+__SYCL_EXPORT void __spirv_ControlBarrier(int32_t Execution, int32_t Memory,
```

Comment:
These are for the old host device support, I'd advise to not touch these as it creates an ABI break and that's only for the host.

Reply:
thanks @Naghasan, reverted the ABI change